memor/session.py

```diff
@@ -572,3 +573,8 @@ def masks(self) -> List[bool]:
     def size(self) -> int:
         """Get the session size in bytes."""
         return self.get_size()
+
+    @property
+    def warnings(self) -> Dict[str, Union[float, bool]]:
```

Comment:
same for this.

Reply:
fixed in https://github.com/openscilab/memor/pull/172/commits/2d2fde1eed993217d3ca1b2c7db0886aa5ab4425